src/main/java/com/buildmaven/add/Test.java

```diff
@@ -0,0 +1,9 @@
+package com.buildmaven.add;
+
+public class Test {
+
+	public static void main(String[] args) {
+		
```

Comment:
Please use standard comments for method & class level @rahulstar7 
As per discussion with @gdpatil39  plz resove this issue

Reply:
done